org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/window/WRecordInfo.java

```diff
@@ -481,7 +481,7 @@ private void addLine (int AD_Column_ID, Timestamp Updated, int UpdatedBy,
 		Vector<String> line = new Vector<String>();
 		//	Column
 		MColumn column = MColumn.get (Env.getCtx(), AD_Column_ID);
-		line.add(Msg.translate(Env.getCtx(), column.getColumnName()));
+		line.add(Msg.getElement(Env.getCtx(), column.getColumnName()));

```

Comment:
I think we should replace other `translate` in this class with `getElement` call as well.

Reply:
done